integration-tests/gradle/src/test/resources/basic-java-native-module/build.gradle

```diff
@@ -27,7 +27,7 @@ compileTestJava {
 
 quarkusBuild {
     nativeArgs {
-        graalvmHome="/foo/bar"
         containerBuild=true
+        builderImage="quay.io/quarkus/ubi-quarkus-native-image:19.3.1-java11"
```

Comment:
Is this the only way to configure it? Couldn't we propagate the property from the Maven process?